UI/Selenium/Steps/WaitingRoomPageSteps.cs

```diff
@@ -66,7 +78,7 @@ public void CheckParticipantsAreInWaitingRoom()
         {
             foreach (var participant in _hearing.Participant)
             {
-                if (!participant.Party.Name.ToLower().Contains("judge"))
+                if (!(participant.Party.Name.ToLower().Contains("judge") || participant.Party.Name.ToLower().Contains("panel")))
```

Comment:
This might be wrong.  There are also Judicial office holders and wingers to include when you have more that one type of judge.